src/languages/r.js

```diff
@@ -85,18 +85,17 @@ export default function(hljs) {
               // doctags shouldn’t be treated as such. See
               // `test/markup/r/roxygen.txt` for an example.
               scope: 'doctag',
-              begin: '@examples',
+              match: '@examples',
               starts: {
                 contains: [
-                  { begin: /\n/ },
                   {
-                    begin: /#'\s*(?=@[a-zA-Z]+)/,
-                    endsParent: true,
+                    match: /\n#'\s*(?=@[a-zA-Z]+)/,
+                    endsParent: true
                   },
                   {
-                    begin: /#'/,
+                    begin: /\n#'/,
```

Comment:
This looks weird, why not `^`?

Reply:
The intent is to match the line break itself, not the start of the line. This is supposed to be a match across multiple lines, causing the parent mode to continue until the terminating condition is met (either explicitly via `endsParent: true`, or implicitly after a line break not followed by `#'`). Using `^` doesn’t do that.

However, I don’t seem to actually require `excludeBegin: true` — the tests stay green after removing it.

… There might be another, clever way of telling hl.js to continue the parent mode onto the next line but I haven’t been able to figure out how.

Reply:
Add a separate rule to match `\n` with a lookahead for `#`... less magical and you can commit what it's doing to make it clear.?

Reply:
Ah, I think I got you. How’s 63760293270a6320e898ac2dedededa8fbbcada0?